app/views/admin/qualification_report_uploads/new.html.erb

```diff
@@ -7,6 +7,13 @@
     <%= form_with url: admin_qualification_report_uploads_path, multipart: true do |f| %>
       <div class="govuk-form-group">
         <%= f.label 'file', "Upload a CSV file", class: "govuk-label" %>
+
+        <% @dqt_report_consumer&.errors&.each do |error| %>
```

Comment:
The double safe-navigations look a bit stinky. Something like this or wrap it in an `if`?:
```suggestion
        <% @dqt_report_consumer && @dqt_report_consumer.errors.each do |error| %>
```